tests/test_cli.py

```diff
@@ -263,6 +318,82 @@ def test_cli_start_with_input(runner):
                               'mcr.microsoft.com/azureiotedge-testing-utility:1.0.0-rc1'])
 
 
+@pytest.mark.skipif(get_docker_os_type() == 'windows', reason='The base image of edgeHubDev image is 1809 but agent is 1803.'
+                    'So it does not support windows container.')
+def test_cli_start_with_create_options_for_bind(runner):
+    test_resources_dir = os.path.join('tests', 'test_compose_resources')
+    try:
+        if get_docker_os_type() == "windows":
+            deployment_json_file_path = os.path.join(test_resources_dir, 'deployment_with_create_options_for_bind.json')
+            update_file_content(deployment_json_file_path, '/usr:/home/moduleuser/test',
```

Comment:
instead of  change the content in same file. how about just copy to another windows specific file and replace?

Reply:
I don't want to maintain many config files for one scenario. I think it's better to use one config file for one scnario.

Reply:
Updated code